src/org_parser/render.cljc

```diff
@@ -18,10 +20,17 @@
 ;; to be extenden to the full feature scope.
 
 
+(defn- serialize-text
```

Comment:
@munen How does these multi-arity functions work with this keyworded lists? The input for this function can be something like `[:text [:text-normal "uiea"] [:text-bold "foo"]]`